src/Tasks.js

```diff
@@ -0,0 +1,33 @@
+export default class Tasks {
+  static index = 0;
+
+  constructor() {
+    this.tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+  }
+
+  addTask = (description, completed) => {
+    this.tasks.push({
+      description,
+      completed,
+      index: parseInt(Tasks.index, 10),
+    });
+  };
```

Comment:
- [ ] When I add a task the property `index` should be set to the value of the new array (i.e. if a 5th task is added to the list, the `index` of that task should equal to 5) but currently when I check your application all indexes are 0 please address that
![idx](https://user-images.githubusercontent.com/44745855/165818057-6d0e031e-3f54-4566-be9b-3e3d8d7cf7ac.png)